base/ltluatex.dtx

```diff
@@ -23,16 +23,14 @@
 %<tex>  \ProvidesFile\undefined\begingroup\def\ProvidesFile
 %<tex>  #1#2[#3]{\endgroup\immediate\write-1{File: #1 #3}}
 %<tex>\fi
-%<plain>\ProvidesFile{ltluatex.tex}%
+%<plain>\ProvidesFile{ltluatex.tex}[2024/02/11 v1.2c LuaTeX support for plain TeX (core)]
 %<*driver>
-\ProvidesFile{ltluatex.dtx}
+% \fi
+\ProvidesFile{ltluatex.dtx}[2024/02/11 v1.2c LuaTeX Kernel (LuaTeX support)]
```

Comment:
again the date in 2 places (that would always be the same). This is one of the reaons why I made macros for the date.

Reply:
rearranged docstrip guards so date only given once